engines/bops_api/app/services/bops_api/application/creation_service.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+module BopsApi
+  module Application
+    class CreationService
+      class CreateError < StandardError; end
+
+      def initialize(**options)
+        raise CreateError, "This service is not permitted in production" if Bops.env.production?
```

Comment:
If this error might be seen by end users then referring to a 'service' might not make sense. Maybe e.g.:

```suggestion
        raise CreateError, "Creating planning applications by this method is not permitted in production" if Bops.env.production?
```

Reply:
An end user wouldn't ever see this on the frontend and we'd get an error come through on AppSignal. We'd send a response back to PlanX in an error json so we can still make the message clearer I think

Reply:
Update the message and added a spec for this too